### DISCUSSION TITLE
What the hello is hello?

### DISCUSSION THREAD

**Discussion by soundandform** [helloSystem/hello#335] [Q&A]
I’m a reasonably advanced computer user (40+ years), programmer and electrical engineer.  I have looked at the hello website and guide a number of times in recent history.  And, I *still* cannot tell what this project is exactly. The front page documentation needs to be a lot clearer.  Is it a full blown operating system?  Is it some sort of weird window manager for linux?  Is it a compatibility layer or emulator of some sort?

It’s called a “desktop system”.  What is a desktop system?  This is not standard terminology.  The lack of "operating" between "desktop" and "system" seems to imply something, but I can't figure out what that is.

It’s intriguing.  And, I want it to be successful — whatever it is!   But, if a very technically savvy person can’t decipher your front page, I don’t know how any regular user will.

Seriously I’m still confused. I now notice “Debian” in that screenshot.  Does it run Linux GUI applications?  Does it run MacOS applications?  WHAT IS HELLO??  I must know.  Please just put this information in the first paragraph so I can stop coming back here every month.

**Comment by probonopd** [comment]
https://hellosystem.github.io/docs/

> helloSystem is a desktop system for creators with a focus on simplicity, elegance, and usability. Its design follows the “Less, but better” philosophy. It is intended as a system for “mere mortals”, welcoming to switchers from the Mac. [FreeBSD](https://www.freebsd.org/) is used as the core operating system. Please refer to https://github.com/helloSystem/hello if you would like to learn more about the ideas and principles behind hello.

> Is it a full blown operating system?

Yes.

> Is it some sort of weird window manager for linux?

No.

> Is it a compatibility layer or emulator of some sort?

No. But it contains the Linuxulator, which can be used to run applications compiled for Linux if native ones for FreeBSD are not available (e.g., using the aforementioned Debian runtime).

**Comment by soundandform** [comment]
OK, thanks for the clarification. Why the aversion to calling it an operating system?  Everyone knows what an OS is. My mother knows what an OS is! A "desktop system" sounds like something that's sold at IKEA. This is confusing, unnecessary and seriously muddling the marketing of what seems like a great project.

**Reply by larshenrikoern** [reply]
Hi
It is a desktop based on qt running on top of FreeBSD operating system. It has all the features of FreeBSD. Although some of them are disabled on Hello (among others virtual terminals).

Have a nice day

**Reply by probonopd** [reply]
It's really the FreeBSD operating system preconfigured for desktop usage.
